### ISSUE TITLE
Can no longer use admin.ping

### ISSUE THREAD

**Issue by fresheneesz** [kissjs/node-mongoskin#177]
After upgrading from v1.4.4 to 1.4.13 and 2.0.0, I can no longer use the `db.admin().ping` method. Its undefined. In fact, it look like lots of methods aren't available in there, and there's a weird circular reference to `admin`. Maybe the wrong object is being returned for `admin`?

**Comment by fresheneesz** [comment]
Just confirmed that ping works properly in v1.4.4

**Comment by vkarpov15** [comment]
Yeah you're right, it needs to be `db.admin()` rather than `db.db('admin');`

**Comment by fresheneesz** [comment]
@vkarpov15 Thanks for the quick response! However, I updated and still have the same issue. I tried debugging through to that point and checking `db.admin().ping`, but it was still undefined. I tried a bit to figure out why that might be but I couldn't find it. 

I noticed tho, that admin.js doesn't seem to have the same thing as in this repo. I installed the latest versio nof mongoskin (2.0.1), and I see the following file:
```
/*!
 * mongoskin - admin.js
 *
 * Copyright(c) 2011 - 2012 kissjs.org
 * Copyright(c) 2012 fengmk2 <fengmk2@gmail.com>
 * MIT Licensed
 */

"use strict";

var SkinAdmin = exports.SkinAdmin = function(db) {
  return db._admin();
}
```

Note the underscore in `db._admin()`. I tried changing it to `db.admin()` but got a stack overflow.
